images/installer/Dockerfile

```diff
@@ -22,7 +22,7 @@ USER root
 # configurations for the two images.
 RUN mkdir -p /usr/share/ansible/ && ln -s /opt/app-root/src /usr/share/ansible/openshift-ansible
 
-RUN INSTALL_PKGS="skopeo" && \
+RUN INSTALL_PKGS="skopeo openssl java-1.8.0-openjdk-headless python-passlib httpd-tools" && \
```

Comment:
sorry should have been a bit more explicit: `requirements.txt` is in the TLD of the repo, and the suggestion here is to add this line there:

`passlib==1.6.5`

instead of installing `python-passlib` with yum. Using 1.6.5 as the version here as this is what the current el7 RPMs provide (same criteria used e.g. for the `boto` dependency).